docs/source/tutorial.rst

```diff
@@ -117,39 +123,73 @@ build, run the following inside vagrant:
 
   $ /project/sawtooth-core/bin/build_all
 
-Running validator
-=================
+Validator Start-up Process
+==========================
+
+Create Genesis Block
+--------------------
+
+The first time that a validator is run, the validator needs to ingest a batch
+of transactions that set up some initial values for use by the first block in
+the block-chain, called the "genesis block."
+
```

Comment:
This may be misinterpreted to mean that each validator must generate a genesis block. Usually starting a validator it just needs to know where to find the distributed ledger network. As a developer you may often be creating short lived test networks. In that case you need to create the genesis block.